### DISCUSSION TITLE
本地搭建move环境运行找不到依赖

### DISCUSSION THREAD

**Discussion by Fdango** [NonceGeek/Move-Camp#305] [Q&A]
Move.toml

[package]
name = "demo"
version = "0.0.0"

[dependencies]
MoveNursery = { git = "https://github.com/move-language/move.git", subdir="language/move-stdlib/nursery", rev = "main"}

[addresses]
std =  "0x1"

sources -> first.move
```

script{
  use 0x1::Debug;  // import other modules, 0x1 is an offical modules library.
  fun main(){
    let num:u64 = 1024;
    Debug::print(&num);
  }
}
```

执行move build之后报错：
<img width="788" alt="image" src="https://user-images.githubusercontent.com/17996704/220081131-d0494f8b-a62c-4ae4-8bd7-87a2f2417196.png">

**Comment by qiwihui** [comment]
用 `use 0x1::debug;` 可以解决

**Reply by Fdango** [reply]
完美解决
